nestkernel/conn_builder.h

```diff
@@ -87,6 +87,9 @@ class ConnBuilder
   //! Create connection between given nodes, fill parameter values
   void single_connect_( index, Node&, thread, librandom::RngPtr& );
 
+  //! Create connection between given nodes, fill parameter values
+  void skip_conn_parameter_( thread, librandom::RngPtr& );
```

Comment:
Please write more explicit documentation. I believe, the `skip_conn_parameter_` does not create connections, but propagates `weight` and `delay` parameters in ParameterArrays.